### PR TITLE
Complex authorization

### PR DESCRIPTION
**Pull request by dimaninc**

### CODE REVIEW

File: src/Uploadcare/Api.php [hunk 1]
```diff
@@ -305,11 +308,13 @@ private function __getPath($type, $params = array())
    *
    * @param resource $ch. Curl resource.
    * @param string $type Request type. Options: get, post, put, delete.
-   * @throws Exception
+   * @throws \Exception
    * @return void
    */
   private function __setRequestType($ch, $type = 'GET')
   {
+	$this->current_method = strtoupper($type);
```
Comment (dmitry-mukhin) [inline src/Uploadcare/Api.php]: ahem

Comment (dimaninc) [inline src/Uploadcare/Api.php]: for 100% probability of method name is in upper case

Comment (dmitry-mukhin) [inline src/Uploadcare/Api.php]: я про таб

Comment (dimaninc) [inline src/Uploadcare/Api.php]: fixed

**Pull request merged**